specification/wse/src/main/scripts/org/kaazing/specification/wse/downstream/binary/subsequent.request.out.of.order/response.rpt

```diff
@@ -37,64 +35,13 @@ write header "Content-Type" "application/octet-stream"
 write header "Connection" "close"
 write flush
 
-write await FIRST_PING_RECEIVED
-write [0x8A 0x00]
-
-write await SECOND_PING_RECEIVED
-write [0x8A 0x00]
-
 write await OUT_OF_ORDER_REQUEST_RECEIVED
 write close
 
-
-# Upstream
-accept http://localhost:8080/path/QD12TQhXT6I5KeCrDe6g0IC9qoDkIioh?.krn=1
-accepted
-connected
-
-read await CREATED
-read method "POST"
-read version "HTTP/1.1"
-read header "Content-Type" "application/octet-stream"
-read header "X-Sequence-No" ${wse:asString(sequence + 1)}
-
-read [0x89 0x00]
-read [0x01 0x30 0x31 0xFF]
-read closed
-read notify FIRST_PING_RECEIVED
-
-write status "200" "OK"
-write version "HTTP/1.1"
-write header content-length
-write close
-
-# Upstream
-accept http://localhost:8080/path/QD12TQhXT6I5KeCrDe6g0IC9qoDkIioh?.krn=2
-accepted
-connected
-
-read await CREATED
-read method "POST"
-read version "HTTP/1.1"
-read header "Content-Type" "application/octet-stream"
-read header "X-Sequence-No" ${wse:asString(sequence + 2)}
-
-read [0x89 0x00]
-read [0x01 0x30 0x31 0xFF]
-read closed
-read notify SECOND_PING_RECEIVED
-
-write status "200" "OK"
-write version "HTTP/1.1"
-write header content-length
-write close
-
```

Comment:
I realized there is an opportunity to make the test script tighter. The test is just supposed to verify if the server script will reject the downstream request if it is out of order.